### PR TITLE
[cisco_meraki] Add event.action and message to specific events

### LINKED ISSUE #7790: [cisco_meraki] Add event.action and message to specific events
# Integration release checklist

Add `event.action` to: `cisco_meraki.event_subtype`: `multiple_dhcp_servers_detected`

Keep important information in `message` field for `cisco_meraki.event_subtype`:
- dhcp
- client_vpn_connect
- blocked
- port

### All changes

- [X] Change follows the [contributing guidelines](https://github.com/elastic/integrations/blob/main/CONTRIBUTING.md)
- [X] Supported versions of the monitoring target are documented
- [X] Integration or [System tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/system_testing.md) exist
- [X] Documentation exists, [useful guidelines](https://github.com/elastic/integrations/blob/main/docs/documentation_guidelines.md) to follow
- [X] Fields follow [ECS](https://github.com/elastic/ecs) and [naming conventions](https://www.elastic.co/guide/en/beats/devguide/master/event-conventions.html)
- [X] At least a manual test with ES / Kibana / Agent has been performed.
- [X] Required Kibana version set to: 8.0.0

### PR DESCRIPTION
**Pull request by LaZyDK**
## What does this PR do?

Add `event.action` to: `cisco_meraki.event_subtype`: `multiple_dhcp_servers_detected`

Keep important information in `message` field for `cisco_meraki.event_subtype`:
- dhcp
- client_vpn_connect
- blocked
- port

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Closes #7790
- Requires #7771

### CODE REVIEW

File: packages/cisco_meraki/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.13.1"
+  changes:
+    - description: Add event.action and message to specific events.
+      type: bugfix
```
Comment (efd6) [inline packages/cisco_meraki/changelog.yml:5]: Bugfix or enhancement?

Comment (LaZyDK) [reply packages/cisco_meraki/changelog.yml]: Changed to enhancement

**Comment by LaZyDK** [comment]
Ready for another test

**Comment by LaZyDK** [comment]
Fixed conflicts. Ready again.

**Review by efd6 — APPROVED** [review]

**Pull request merged**